lib/auth/init.go

```diff
@@ -680,11 +681,11 @@ func initCluster(ctx context.Context, cfg InitConfig, asrv *Server) error {
 		}
 		span.AddEvent("completed creating database object import rules")
 
-		span.AddEvent("creating preset health check config")
-		if err := createPresetHealthCheckConfig(ctx, asrv); err != nil {
+		span.AddEvent("creating preset health check configs")
+		if err := createPresetHealthCheckConfigs(ctx, asrv); err != nil {
 			return trace.Wrap(err)
```

Comment:
nit: since we removed this from the individual errs below

```suggestion
			return trace.Wrap(err, "failed to create preset health_check_config resources")
```